db/column_family.cc

```diff
@@ -477,13 +477,16 @@ void SuperVersion::Cleanup() {
   cfd->UnrefAndTryDelete();
 }
 
-void SuperVersion::Init(ColumnFamilyData* new_cfd, MemTable* new_mem,
-                        MemTableListVersion* new_imm, Version* new_current) {
+void SuperVersion::Init(
+    ColumnFamilyData* new_cfd, MemTable* new_mem, MemTableListVersion* new_imm,
+    Version* new_current,
+    std::shared_ptr<const SeqnoToTimeMapping>&& new_seqno_to_time_mapping) {
```

Comment:
Even when always saving a copy, shared_ptr without `&&` is recommended. This is a case where I agree with MSVC: https://learn.microsoft.com/en-us/cpp/code-quality/c26416?view=msvc-170

Reply:
Thank you for the pointer!